### PR TITLE
Meta: Drop local release dependencies

### PR DESCRIPTION
**Pull request by fregante**
They're only used on GitHub Actions. If you use `npm run start`, install `web-ext` globally.

### CODE REVIEW

**Comment by yakov116** [comment]
How is your package.lock change so big?

**Comment by fregante** [comment]
https://npm.broofa.com/?q=web-ext 

We just have a buttload of dev dependencies

**Comment by yakov116** [comment]
OMG

**Comment by yakov116** [comment]
> If you use `npm run start`, install `web-ext` globally.

https://github.com/sindresorhus/refined-github/blob/main/contributing.md#L109

Should get a comment here

File: contributing.md [hunk 1]
```diff
@@ -113,4 +113,6 @@ npm run start # Open extension in Chrome
 npm run start:firefox # Open extension in Firefox
 ```
 
+Note: They both require [web-ext](https://github.com/mozilla/web-ext) to be installed globally. Run `npm i -g web-ext`
```
Comment (yakov116) [inline contributing.md:116]:
```suggestion
**Note**: They both require [web-ext](https://github.com/mozilla/web-ext) to be installed globally. Run `npm i -g web-ext`
```

**Review by yakov116 — APPROVED** [review]
Also removes almost 100MB from install size! (At least for me)

**Pull request merged**